ocf_datapipes/utils/utils.py

```diff
@@ -19,6 +20,28 @@
 logger = logging.getLogger(__name__)
 
 
+def return_sys_idx_with_cont_nan(arr: np.ndarray) -> np.ndarray:
+    """
+    Returns indexes of system id's in which if they have
+    contigous 289 NaN's
+    """
+    count, max_count = 0, 0
+    number_of_sys = arr.shape[1]
+    sys_idx = []
+    for i in range(0, number_of_sys):
+        array = arr[:, i]
+        for e in array:
```

Comment:
Could we do this in a numpy call, as in get the counts of nans, in something like this

```suggestion
        np.count_nonzero(np.isnan(array))
```

Reply:
The suggested method counts all the `NaN's` in the array, but we need the count of contiguous `NaN's` in the array i.e 288 to drop the system, I'll think of a better workaround

Reply:
There are some ideas here which might be helpful https://stackoverflow.com/questions/41721674/find-consecutive-repeated-nan-in-a-numpy-array

Reply:
Thanks, incorporated the code!